cmd/topology-aware/nri-resmgr-topology-aware-deployment.yaml.in

```diff
@@ -136,6 +143,8 @@ data:
         CPU: 750m
     logger:
       Debug: nri-resmgr,resource-manager,cache,policy
+    instrumentation:
+      HTTPEndpoint: ":8891"
```

Comment:
This probably shouldn't be `instrumentation/HTTPEndpoint` any more but out/up to reflect the package having been moved out from `instrumentation`.

Reply:
Is it going to pass ?